src/manageiq_client/api.py

```diff
@@ -115,6 +115,16 @@ def delete(self, url, **payload):
                 raise APIException("JSONDecodeError: {}".format(data.text))
         return self._result_processor(data)
 
+    def options(self, url, **opt_params):
+        self.logger.info("[RESTAPI] OPTIONS %s %r", url, opt_params)
+        data = self._sending_request(
+            partial(self._session.options, url, params=opt_params, verify=False))
```

Comment:
why are you passing `verify=False` here?
ssl verification is determined in the session level (https://github.com/ManageIQ/manageiq-api-client-python/blob/master/src/manageiq_client/api.py#L34), not specific method call